masq/src/commands/set_exit_location_command.rs

```diff
@@ -0,0 +1,224 @@
+    that is, if there is no exit Node available in any of your preferred countries, you'll get an error. However, \
+    if you just want to make a suggestion, and you don't mind Nodes in other countries being used if nothing is available \
+    in your preferred countries, you can specify --fallback-routing, and you'll get no error unless there are no exit Nodes \
+    available anywhere.\n\n\
+    Here are some example commands:\n\
+        masq> exit-location                                                       // disable exit-location \n\
+        masq> exit-location --fallback-routing                                    // disable exit-location \n\n\
+        masq> exit-location --country-codes \"CZ,PL|SK\" --fallback-routing       // fallback-routing is ON, \"CZ\" and \"PL\" countries has same priority \"1\", \"SK\" has prirority \"2\"\n\
+        masq> exit-location --country-codes \"CZ|SK\"                             // fallback-routing is OFF, \"CZ\" and \"SK\" countries has different prirority\n";
+
+// TODO update following help when GH-469 is done
+const COUNTRY_CODES_HELP: &str = "Establish a set of countries that your Node should try to use for exit Nodes. You should choose from the countries that host the \
+        Nodes in your Neighborhood. List the countries in order of preference, separated by vertical pipes (|). If your level of preference \
+        for a group of countries is the same, separate those countries by commas (,).\n\
+        To obtain codes you cant use 'country-codes-list' command. You can specify country codes followingly:\n\n\
+        masq> exit-location --country-codes \"CZ,PL|SK\"                          // \"CZ\" and \"PL\" countries has same priority \"1\", \"SK\" has prirority \"2\" \n\
+        masq> exit-location --country-codes \"CZ|SK\"                             // \"CZ\" and \"SK\" countries has different prirority\n";
+
+const FALLBACK_ROUTING_HELP: &str = "If you just want to make a suggestion, and you don't mind Nodes in other countries being used if nothing is available \
+     in your preferred countries, you can specify --fallback-routing, and you'll get no error unless there are no exit Nodes \
+     available anywhere. \n Here are some examples: \n\n\
+     masq> exit-location --country-codes \"CZ\" --fallback-routing              // Set exit-location for \"CZ\" country with fallback-routing on \n\
+     masq> exit-location --country-codes \"CZ\"                                 // Set exit-location for \"CZ\" country with fallback-routing off \n";
+
+pub fn set_exit_location_subcommand() -> App<'static, 'static> {
+    SubCommand::with_name("exit-location")
+        .about(EXIT_LOCATION_ABOUT)
+        .arg(
+            Arg::with_name("country-codes")
+                .long("country-codes")
+                .value_name("COUNTRY-CODES")
+                .validator(common_validators::validate_exit_locations)
+                .help(COUNTRY_CODES_HELP)
+                .required(false),
```

Comment:
The way you're currently doing this, you should probably be specifying minimum and maximum value counts here. Since there has to be one value, and there can't be any more than one value, both minimum and maximum should be 1. Then `clap` will take care of more syntax validation, and you won't have to.

Reply:
Finally I used .value_delimiter("|") and there could be maximum c. 250 countries, but I do not thing there should be limit. Minimum values could be 0 to unset all exit-location preferences